node/plugin/cmd/main.go

```diff
@@ -146,6 +146,14 @@ func pluginOps(ctx *cli.Context) {
 			return
 		}
 		log.Printf("Info: successfully updated quorums, for operator ID: %x, operator address: %x, socket: %s, and quorums: %v", operatorID, sk.Address, config.Socket, config.QuorumIDList)
+	} else if config.Operation == "update-socket" {
```

Comment:
I think metrics and logs should also be updated as we do in https://github.com/Layr-Labs/eigenda/blob/master/node/node.go#L362-L363

Reply:
umm. the issue here is - this is not running at the node level, it's a separate script. We don't interact with node as of now where the metrics server is. I think this should be okay without metrics as the operator is executing this so they  know what they are doing.

also logs are already there